apps/omg_watcher/test/support/integration/bad_child_chain_block.ex

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.Integration.BadChildChainBLock do
+  @moduledoc false
```

Comment:
definitely requires a `moduledoc`, but generally a big :+1: for the DRYing here

Reply:
" Module useful for creating integration tests where we want to simulate bad child chain
    which is returning a bad block for a particular block number. "